### DISCUSSION TITLE
Loss isn't converging

### DISCUSSION THREAD

**Discussion by zhaowenZhou** [mlfoundations/open_clip#874] [Q&A]
I am currently training CLIP model on my own dataset.
For image part, all images come from different set of sequential images and for text part, the title is scene, weather, light, for example: 'urban, sunny, backlight'.
![image](https://github.com/mlfoundations/open_clip/assets/90346002/b0c7d393-6d4f-4c57-a50d-f8ffccbdcd93)
The loss isnt converging
![image](https://github.com/mlfoundations/open_clip/assets/90346002/246d88b6-fcc2-4d74-8964-9dd2e2dbfe8b)
![image](https://github.com/mlfoundations/open_clip/assets/90346002/951ba86c-836e-4ac0-b4b0-d85e802d0022)
As for now i think the problem is that some of the images within a batch are from the same sequence, images will be alike and are likely to have same label. But model will still try to contrast it, make it hard to learn any useful information.
Any advice or idea will be greatly appreciated

**Comment by gongjizhang** [comment]
Iim encountering a similar issue where the loss stops decreasing, specifically reaching around ln(0.5). My dataset consists of two classes。
